service/controller/v22/resource/bridgezone/resource.go

```diff
@@ -184,10 +187,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 
 		r.logger.LogCtx(ctx, "level", "debug", "message", "got intermediate zone ID")
```

Comment:
This is a good example where `action` key would be useful. That would allow to group intertwined logs from concurrently executed code.